scipy/ndimage/tests/test_measurements.py

```diff
@@ -810,6 +811,7 @@ def test_maximum03(xp):
         assert_almost_equal(output, xp.asarray(4.0), check_0d=False)
 
 
+@skip_xp_backends('dask.array', reason="no argsort in Dask")
 def test_maximum04(xp):
```

Comment:
Ditto with [`maximum` in dask-image]( https://github.com/dask/dask-image/blob/77a29ba78db608c5fe21f9f51745ec4bd01d5681/dask_image/ndmeasure/__init__.py#L466 )